tasks/util/browserify_wrapper.js

```diff
@@ -71,13 +72,24 @@ module.exports = function _bundle(pathToIndex, pathToBundle, opts, cb) {
             });
     }
 
-    bundleStream
-        .pipe(applyDerequire())
-        .pipe(fs.createWriteStream(pathToBundle))
-        .on('finish', function() {
-            logger(pathToBundle);
-            done();
-        });
+    if(sourceMap) {
+        bundleStream
+            .pipe(applyDerequire())
+            .pipe(exorcist(sourceMap))
```

Comment:
This is generating the sourcemap of the unminified bundle. The real value is in a sourcemap of the minified bundle (which, for clarity, gets extension `.min.js.map`), then you never need the unminified bundle.

Reply:
Good call. Addressed in https://github.com/plotly/plotly.js/pull/5527/commits/7d0145ae317ecc9793c1b2975386cefe122b75b7.